circle.yml

```diff
@@ -1,23 +0,0 @@
-machine:
-  node:
-    version: 6
-
-dependencies:
-  pre:
-    - bundle install
-    # https://discuss.circleci.com/t/using-the-latest-version-of-chrome-on-circleci/871/4
-    - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-    - sudo dpkg -i google-chrome.deb
-    - sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
-    - rm google-chrome.deb
-
-test:
-  pre:
-    - gulp -v
-    - scss-lint -v
```

Comment:
Huh, was the old CircleCI config actually running `scss-lint` _twice_, once here and once as part of `npm test` below at line 21?

Reply:
yeah 😢